Overview.bs

```diff
@@ -368,8 +370,23 @@ not by wall-clock time, but by the progress of scrolling in a [=scroll container
 <div class="constructors">
 
 :   <dfn constructor for=ScrollTimeline lt="ScrollTimeline(options)">ScrollTimeline(options)</dfn>
-::  Creates a new {{ScrollTimeline}} object. The attributes of the {{ScrollTimeline}} are 
-    initialized with the respective members of the {{options}} dictionary.
+::  Creates a new {{ScrollTimeline}} object using the following procedure:
+
+    1. Let <var>timeline</var> be a new {{ScrollTimeline}} object.
```

Comment:
(For what it's worth, I think bikeshed now allows `|timeline|` as a shorthand for specifying variables. I think Tab might also recommend that as being more readable so going forward we should probably prefer that style.)

Reply:
Done.